src/sendLike.js

```diff
@@ -0,0 +1,12 @@
+const sendLike = async (objName) => {
+  /* eslint-disable-next-line */
+  const response = await fetch('https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/WeWaTSWdgI5EoHhdFxch/likes/', {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({ item_id: objName }),
+  });
+};
+
+export default sendLike;
```

Comment:
- [ ] When I click the like part, doesn't do anything, I observe you have a like button on the reservation, but this button would be better on the home page for every pokémon. Please fix this. 🐱 
- This is a good part to add it (make it clickable):
![image](https://user-images.githubusercontent.com/20563252/132966140-331f5e39-309e-4aaf-addd-616b74f8d348.png)

Reply:
Are you there?